code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -49,6 +49,11 @@
 			icon = 'icons/mob/alien.dmi'
 			icon_state = "AlienMMI"
 			alien = 1
+		else if(istype(O,/obj/item/organ/internal/brain/drask))

```

Comment:
Give brains an `mmi_icon` var, which is used in MMI code to display how they look